lua/nvim-treesitter/indent.lua

```diff
@@ -5,7 +5,6 @@ local tsutils = require "nvim-treesitter.ts_utils"
 local M = {}
 
 M.avoid_force_reparsing = {
-  python = true,
```

Comment:
this will make indentation after `\` stop to work

Reply:
before you would have strange indentation on starting a new line using `<cr>` in insert mode. E.g. indenting a aligned_indent though the parens are already closed.